rfc/rfc-2/Entity-key-encoding-for-kv-store-deprecated.md

```diff
@@ -17,13 +17,19 @@
   under the License.
 -->
 
-# RFC-1: Entity Key Encoding design for KV store
+# RFC-1: Entity Key Encoding design for KV store (removed)
```

Comment:
@pithecuse527 can we just revert the file name (remove "-deprecated")? Thanks a lot.

Reply:
My bad. Just reverted.